src/helpers/print.js

```diff
@@ -3,7 +3,7 @@ import i18n from 'i18next';
 import actions from 'actions';
 import core from 'core';
 
-export default (dispatch, isEmbedPrintSupported) => {
+export default (dispatch, isEmbedPrintSupported) =>  {
```

Comment:
Hmmm what happened here?

Reply:
was it supposed to be removed? i think i must have had some merge conflict problems and I might have resolved it incorrectly ?

Reply:
as discussed. it might have been a bad resolving of a merge conflict. i have reverted it back to the original.